strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/support/ArtifactEntrySearchCriteria.java

```diff
@@ -0,0 +1,63 @@
+package org.carlspring.strongbox.services.support;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+public class ArtifactEntrySearchCriteria
+{
+
+    private Integer uselessnessDays;
```

Comment:
Yeah... I guess the name should be something like `lastAccessedTimeInDays`, or... maybe something else along these lines?

Reply:
Much better :)